server/fleet/software_installer.go

```diff
@@ -339,8 +339,9 @@ type SoftwarePackageOrApp struct {
 // HostSoftwareInstall represents installation of software on a host from a
 // Fleet software installer.
 type HostSoftwareInstall struct {
-	InstallUUID string    `json:"install_uuid" db:"install_id"`
-	InstalledAt time.Time `json:"installed_at" db:"installed_at"`
+	InstallUUID string    `json:"install_uuid,omitempty"`
+	CommandUUID string    `json:"command_uuid,omitempty"`
```

Comment:
Should we have two different structs? I'm a little concerned about the maintainability of this polymorphism and finding ourselves in a situation down the road like we did with host MDM info where sometimes fields are populated and sometimes they aren't. If we need to move quickly, we can go with this for now and maybe add a todo to deal with it later.

We should definitely add some code comments to explain the two different fields.

Reply:
@gillespi314 that's a fair concern! I can try out 2 different structs and raise a flag if it seems like it will take longer.

Reply:
Note: we decided to wait till we can confer with Martin and move to 2 different types (if necessary) in a subsequent ticket.